src/components/ui/Card.vue

```diff
@@ -14,10 +14,24 @@ export default {
 .card {
   background-color: var(--white);
   box-shadow: var(--shadowCard);
-  padding: 1em;
+  padding: 0;
   position: relative;
   border-radius: var(--cardRadius);
 }
+
+.card:hover {
+  box-shadow: var(--shadowCardHover);
+}
+
+.card:hover .card__link h2 {
```

Comment:
The bold underline (box-shadow) was done to make it look like the [designs](https://app.zeplin.io/project/5b61ec0e22302659883e6834/screen/5be67996f9d5b53eec29b6ba).